ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/Authorizer.scala

```diff
@@ -324,8 +331,21 @@ object Authorizer {
       ledgerId: String,
       participantId: String,
       errorCodesVersionSwitcher: ErrorCodesVersionSwitcher,
+      userManagementStore: UserManagementStore,
```

Comment:
I can see that apply is only ever used in the tests. Would it not make sense to hide it behind a method called `testAuthorizer` and then to encapsulate the repetitious boiler plate in this method? WDYT?